Sources/ClientRuntime/EventStream/DefaultMessageDecoderStream.swift

```diff
@@ -6,61 +6,65 @@
 //
 
 extension EventStream {
-
+    
```

Comment:
Formatting changes are a result of auto formatting the entire file in xcode via (cmd + shift + i)